src/forms/staff/forms.jsx

```diff
@@ -0,0 +1,126 @@
+import { LoaderCircle } from 'lucide-react';
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from '@/components/ui/dropdown-menu';
+import { PasswordInput } from '@/components/forms/password-input';
+import { specialtiesWithLabels } from '@/utils/utils';
+
+export function RegisterStaffForm({ form, onSubmit, isLoading, error }) {
+
+
+  return (
```

Comment:
Please set up husky and eslint

Reply:
Resolved in https://github.com/FIS2425/frontend/pull/30/commits/6d1f0b655415d6edf33305b6d32024eb938a6152